Mathlib/RingTheory/DedekindDomain/FiniteAdeleRing.lean

```diff
@@ -122,8 +121,7 @@ def Coe.ringHom : RingHom (R_hat R K) (K_hat R K) :=
       -- Porting note (https://github.com/leanprover-community/mathlib4/issues/11041): was `ext p`
       refine funext fun p => ?_
       simp only [Pi.mul_apply, Subring.coe_mul]
-      -- Porting note: added
-      rw [Pi.mul_apply, Pi.mul_apply, Subring.coe_mul] }
+      rfl }
```

Comment:
I guess this is fine, but please double-check.

Reply:
I think this is the same situation as 12 lines above, which was deemed ok.